### ISSUE TITLE
Import from Firefox not available

### ISSUE THREAD

**Issue by RafaelLinux** [zen-browser/desktop#221]
### What feature would you like to see?

I supposed import from Firefox would be available, but it's not. Why?

### What platform are you seeing the problem on?

Linux

### Additional context

Linux AppImage.

**Comment by forteller** [comment]
I had the same issue with the flatpak version.

Thanks!

**Comment by Ari24-cb24** [comment]
Same issue on the Windows version, would benefit a lot from it.

**Comment by mithom** [comment]
> you can sign in with your firefox account once the setup is completed, there's no need for importing

suggestion as i came here to report the same issue: include the option, but give a message saying that you best do this by signing in with your firefox account instead. It eases the user instead instead of frustrating them

**Comment by topongo** [comment]
> > you can sign in with your firefox account once the setup is completed, there's no need for importing
> 
> suggestion as i came here to report the same issue: include the option, but give a message saying that you best do this by signing in with your firefox account instead. It eases the user instead instead of frustrating them

yes, I'd add at least a message for firefox users on how to import data, or even better: a big nice button for logging in at the same point you import data from chrome.

**Comment by Arctomachine** [comment]
Main problem with sync instead of normal import is losing pretty much every session and having to log in everywhere again.
Meanwhile different editions of firefox (normal and dev) can import from each other without major issues.

**Comment by RafaelLinux** [comment]
> you can sign in with your firefox account once the setup is completed, there's no need for importing

I find it very strange that Zen can import from other installed browsers and yet the user is forced to set up Firefox synchronisation in order to be able to import from its ‘parent’ browser.

I don't use the Firefox sync account because I don't care about syncing my computers or having to switch profiles between work and home to avoid syncing bookmarks that don't need to be synced.

Honestly, that solution doesn't make sense to me.

**Comment by brekker23** [comment]
It's very nice that zen supports mozilla accounts because I have firefox on my phone and i can send tabs back and forth between them

**Comment by topongo** [comment]
> Main problem with sync instead of normal import is losing pretty much every session and having to log in everywhere again. Meanwhile different editions of firefox (normal and dev) can import from each other without major issues.

oh wow i always used firefox sync and never imported data directly, I don't exactly like a browser tinkering with password and such of another one...

in my opinion having to log in again is not that big of an hassle, in the name of not moving around critical passwords and cookies, is an acceptable thing

**Comment by mauro-balades** [comment]
This issue should be discussed further. You can view and continue the discussion here: https://github.com/zen-browser/desktop/discussions/881

 ~ This is an automatic operation to better enhancements requests organization.

**Comment by MostHated** [comment]
> you can sign in with your firefox account once the setup is completed, there's no need for importing

I don't login to firefox and only use local profile, so that is not an option for me.

![](https://img.instance.id/mAuRfjo7r9bF/direct)

**Comment by N-Soder** [comment]
I just synced with Firefox since this feature didn't exist, but now it’s applied my Zen themes and settings to Firefox and wiped all my containers and therefore cookies and cache?

I’m guessing this is more of a Firefox Sync 'feature' than directly to do with Zen. Now I have to set up all my containers again and sign back into everything, which is really frustrating when all I wanted to do was try out Zen for an afternoon...

**Comment by ItzRozzii** [comment]
I don't want to sync into my Mozilla account due to the new ToS and it's just sad to me that there is no way to import it without the Sync.

**Comment by ItzRozzii** [comment]
Although a workaround would be that for some of the history you can just open Chrome, take from Firefox, copy from Chrome, delete Chrome's.

**Comment by aditya95sriram** [comment]
I also didn't want to sync into my Mozilla account, but then I realized that because I had been using Firefox sync until now, they already have all the data from me until now. So, I just synced in once from zen to pull all the bookmarks and history etc so far and then signed out. That way, none of my data from this point on will be with Mozilla.

**Comment by matchcase** [comment]
Bumping this, I would like to sync addons, history, bookmarks and logins without using a Mozilla Account

**Comment by friaca** [comment]
Bumping this aswell. I wanted to try Zen but can't import bookmarks, etc. because I don't use a Mozilla account, so for now I can't use Zen as much as I wanted to.

**Comment by genseirin** [comment]
Same here. Creating an account at a 3rd-party service, sync my data there, sync it back to the same device and lastly close that account again is not the same as importing it directly from my local disk.

**Comment by helloitse** [comment]
I exited the setup sequence thinking there might have been an error and not realizing that Zen browser simply isn't setup to import from Firefox. I think this is a true feature request. There are users without a Mozilla account and a growing number of Firefox Users who are trying to get away from Mozilla. 

The existing workaround is to import from Firefox to another browser and then to Zen. A very not zen experience.

**Comment by Game4usYT** [comment]
Bumping this again. i havent found a way to import my data yet, from librewolf browser.

**Comment by digibucc** [comment]
requesting import from firefox. i dont want to log in to sync. this makes no sense.

**Comment by KarmaMarazu** [comment]
> requesting import from firefox. i dont want to log in to sync. this makes no sense.

zen team please add this feature its been years....
